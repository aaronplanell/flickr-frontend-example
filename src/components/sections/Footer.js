import React from 'react';

const Footer = () => (
  <footer>
    <a href="mailto:aaronplanell@gmail.com"
      style={{paddingRight: '5px'}}
      >
        aaronplanell@gmail.com
    </a>
    ···
    <a href="https://github.com/aaronplanell/flickr-frontend-example"
      style={{paddingLeft: '5px'}}
      target='_blank'>
        https://github.com/aaronplanell/flickr-frontend-example
    </a>
  </footer>
);

export default Footer;
