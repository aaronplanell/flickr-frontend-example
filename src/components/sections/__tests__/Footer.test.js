import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Footer from '../Footer';

it('Footer Renders OK', () => {
  const div = document.createElement('div');
  ReactDOM.render(
    <Footer />,
    div);
});

it('Footer Snapshot matchs', () => {
  const tree = renderer.create(
    <Footer />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
