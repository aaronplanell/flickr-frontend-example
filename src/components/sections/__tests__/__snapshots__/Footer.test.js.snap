exports[`test Footer Snapshot matchs 1`] = `
<footer>
  <a
    href="mailto:aaronplanell@gmail.com"
    style={
      Object {
        "paddingRight": "5px",
      }
    }>
    aaronplanell@gmail.com
  </a>
  ···
  <a
    href="https://github.com/aaronplanell/flickr-frontend-example"
    style={
      Object {
        "paddingLeft": "5px",
      }
    }
    target="_blank">
    https://github.com/aaronplanell/flickr-frontend-example
  </a>
</footer>
`;
